bot/src/main/java/de/chojo/gamejam/util/Future.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 DevCord Team and Contributor
+ */
+
+package de.chojo.gamejam.util;
+
+import org.slf4j.Logger;
+
+import java.util.function.BiConsumer;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public final class Future {
+    private static final Logger log = getLogger(Future.class);
+
+    private Future() {
+    }
+
+    public static Object log(Runnable run) {
+        run.run();
+        return null;
```

Comment:
Is there a reason to always return `null` instead of just making the method a `void` one?

Reply:
These are used as quick insertion for the exceptionally call of futures for the whole logging purpose. They require some type of return type